recipe/meta.yaml

```diff
@@ -29,11 +29,8 @@ requirements:
     - gshhg-gmt
     - dcw-gmt
   run:
-    - libblas
-    - libcblas
-    - liblapack
     - {{ pin_compatible('fftw', max_pin='x') }}
-    - {{ pin_compatible('gdal', max_pin='x.x') }}
+    - {{ pin_compatible('libgdal-core', max_pin='x.x') }}
```

Comment:
Switching from `gdal` to `libgdal-core`, because `gdal` contains Python related dependencies that shouldn't be needed in GMT C.